cmd/samples/recipes/sideeffect/sideeffect_workflow.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"context"
+	"github.com/uber-common/cadence-samples/cmd/samples/common"
+	"go.uber.org/cadence/client"
+	"go.uber.org/cadence/worker"
+	"go.uber.org/cadence/workflow"
+	"go.uber.org/zap"
+	"time"
+)
+
+const ApplicationName = "HelloSideEffect"
+
+func SideEffectWorkflow(ctx workflow.Context, value string) error {
+	logger := workflow.GetLogger(ctx)
+	logger.Info("SideEffectWorkflow started")
+	// setup query handler for query type "state"
+
+	err := workflow.SetQueryHandler(ctx, "value", func(input []byte) (string, error) {
+		return value, nil
+	})
+	if err != nil {
+		logger.Info("SetQueryHandler failed: " + err.Error())
+		return err
+	}
+
+	workflow.SideEffect(ctx, func(ctx workflow.Context) interface{} {
+		return value
```

Comment:
Same here. Try to just generate a UUID inside.

Reply:
yes, same here. Changed it.